apache/superset
# feat(ssh_tunnel): Import/Export Databases with SSHTunnel credentials

### SUMMARY

We are now enabling users to export databases that have SSHTunnels related to them. The exported ssh_tunnel will get included in the `database` `yaml` file and will have the used login method (`password` or `private_key + private_key_password`) with masked values.
Then when user tries to import such databases (or any dashboard/chart/dataset/savedquery), the modal will ask the user to enter such credentials.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
1. Export a DB with SSH Tunnel
2. The exported file should include the `ssh_tunnel` with masked login method
3. Try importing a DB that has `ssh_tunnel` in it
4. The modal should ask you to enter the credentials (`password` OR `private_key + private_key_password`)

### ADDITIONAL INFORMATION
